### DISCUSSION TITLE
V3.8.4  SHowInstall-welcome   the -Blockexecution is failing

### DISCUSSION THREAD

**Discussion by ocsscott** [PSAppDeployToolkit/PSAppDeployToolkit#836] [General]
Code

## Notes: No Countdown just defer or continue also -Topmosy keeps it on TOP but tis movable and blocks the app from launching during install

Show-InstallationWelcome -CloseApps 'notepad=NotePad App better English description (please save your data first)' -BlockExecution -AllowDefer -DeferTimes 3 -TopMost $True

screenshows, I click closeapp notepad closes,   but I can relaunch it instantly while the script is still running

Any Ideas?

**Comment by mjr4077au** [comment]
If this is still occurring, please change this discussion into an issue.
